README.en.md

```diff
@@ -494,3 +500,43 @@ as possible, we would need to know information like the followings.
 | Log                  | Error log when an error occurs                                        |
 | Request restriction  | Request to `wovn.global.ssl.fastly.net` with 443 port must be allowed |
 | Using SSI            | Whether you are using SSI(Server Side Includes)                       |
+
+### Wovn Diagnostics Tool
+
+WOVN.php ships with a diagnostics tool that automatically gathers information for debugging purposes. This tool is shippped disabled by default.
+
+This diagnostics tool is meant for clients running WOVN.php with Apache. To enable the Wovn Diagnostics Tool, please do the following tasks:
+
+1. Uncomment the relevant lines in the `.htaccess` files in your documents root *and* the WOVN.php folder.
+
+2. Create a `.htpasswd` file by running the command `htpasswd -cB .htpasswd wovn_diagnostics` inside the WOVN.php folder. This will create a password for username `wovn_diagnostics`.
+
+3. Create an empty `wovnphp_diagnostics.html` file inside the WOVN.php folder.
+
+4. Access the diagnostic tool at `yourwebsite.com/WOVN.php/diagnostics.php`. You will need to login using the password created in step 2.
+
+5. Follow the on screen instructions to get the diagnostic report, which will be written to the `wovnphp_diagnostics.html`.
+
+Please only enable the diagnostics tool when it is necessary to do so.
+
+#### If you are upgrading
+If you are upgrading from an older version of WOVN.php, please manually add the following lines to your `.htaccess` files:
```

Comment:
If this text stays for a while, it is better to state what "older version" means.
```suggestion
If you are upgrading from WOVN.php 1.1.1 or older, please manually add the following lines to your `.htaccess` files:
```